research/quantifying-data-leakage/src/nvflare_gradinv/utils/modify_client_config.py

```diff
@@ -42,12 +43,18 @@ def main():
     config["DATA_ROOT"] = args.data_root
     config["DATASET_JSON"] = args.dataset_json
 
+    if args.prior_file:
+        if "PRIOR_FILE" in config:
+            config["PRIOR_FILE"] = args.prior_file
```

Comment:
prior_file only for client_1/5/9 ("if "PRIOR_FILE" in config"), shall we make the default configs universal and remove this? I think "for i in 1 5 9" should be sufficient?

Reply:
The current script is used to modify the config for both clients which run the inversion code and clients which don't. The two for loops in this [section](https://github.com/holgerroth/NVFlare/tree/gradinv_prior/research/quantifying-data-leakage#cxr_9clients) use different arguments to the script for each use case.